modules/ROOT/pages/cryptography-pgp.adoc

```diff
@@ -17,6 +17,14 @@ This section addresses these scenarios:
 
 This document assumes that you are reasonably familiar with PGP encryption, as well as the concepts of public and private keys and asymmetric cryptography.
 
+== Key Structure
+
```

Comment:
I have not yet encountered "properties" in my editing here and so I am not certain why they are formatted in italic.
Are these variable values? Is so, should they also be in computer/syntax font?
Or is the italic for emphasis? If so, we want to do that rarely.
I don't think we want italic here, but I am not sure how to format properties or algorithms.

Reply:
Please let me know what you decide, so that I can document it in the style ref.